db/statistics.go

```diff
@@ -641,9 +634,10 @@ func WriteValidatorClIcome(day uint64) error {
 		%s
 		on conflict (validatorindex, day) do update set cl_proposer_rewards_gwei = excluded.cl_proposer_rewards_gwei;`,
 			strings.Join(valueStrings, ","))
+		tx, err := WriterDb.Beginx()
```

Comment:
**Nitpick:** The `defer tx.Rollback()` should be right below the `WriterDb.Beginx()`. Right now it is here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/12ff1e60cb8af7c2b9db481ab12b749674158cac/db/statistics.go#L644

Furthermore I think it is weird that the `err` potentially thrown by `WriterDb.Beginx()` is not handled right below `WriterDb.Beginx()` but within the go-routine below. I suggest the following:

```
tx, err := WriterDb.Beginx()
if err != nil {
	return err
}		
defer tx.Rollback()
txList = append(txList, tx)

g.Go(func() error {
	_, err = tx.Exec(stmt, valueArgs...)
```

Reply:
Put it in the Go function because I thought it might save time, but put it now outside